src/components/QuickGrid/QuickGridRowContextActionsHandler.tsx

```diff
@@ -11,6 +11,7 @@ import { DirectionalHint } from '../../index';
 export interface IQuickGridRowAContextActionsHandlerProps {
     onGetRowActions?: (rowIndex: number) => Array<ActionItem>;
     onActionClicked?: (rowIndex: number, action: ActionItem) => void;
+    hideDropdownActionIcons?: boolean;
```

Comment:
i think that you can avoid this property. I mean, here we have a dropdown component that accepts options that may or may not have icons. Why would i just not give the component options without the icons defined?

Reply:
you are right, done :D